src/apiCalls.js

```diff
@@ -0,0 +1,83 @@
+import LeaderBoard from './app.js';
+
+const board = new LeaderBoard();
+export const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUdz/scores/';
+
+// Create game
+export const createGame = (name) => {
+  board.post('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', name)
+    .then((response) => {
+      const container = document.querySelector('.container');
+      const div = document.createElement('div');
+      div.classList.add('message-created');
+      div.innerHTML = `${response.result}`;
+      container.appendChild(div);
+
+      setTimeout(() => {
+        div.classList.add('message-hidden');
+      }, 2000);
+    })
+    .catch((err) => err);
+};
+
+// Get info From Leadboard API
+export const callApi = (url) => {
+  board.get(url)
+    .then((res) => res.result)
```

Comment:
- [ ] After adding a score and clicking the refresh button, the score isn't shown in the list of scores. Please fix it. Check project [the requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Implement%20the%20%22Refresh,to%20the%20API).